src/pyVertexModel/geometry/geo.py

```diff
@@ -754,6 +752,12 @@ def get_num_faces(self, num_cell):
                                  for c_face in self.Cells[num_cell].Faces])
         return num_faces_bottom, num_faces_lateral, num_faces_top
 
+        if c_cell.AliveStatus == 0:
+            # Apply a small shift for debris cells
+            c_cell.Y += 0.1 * (self.XgTop - self.XgBottom)  # Adjust the factor as needed
+
+        self.Cells[cc].Faces = self.Cells[cc].Faces[:len(neigh_nodes)]
+
```

Comment:
Be careful with these changes. This would affect the behaviour of the mechanics. Also, XgTop, and XgBottom are ids of the ghost nodes.

Reject